src/components/newsItem.scss

```diff
@@ -6,6 +6,26 @@
   margin-right: 48px;
   width: 367px;
   transition: width 0.2s ease-in-out, transform 0.2s ease-out;
+  text-decoration: none;
+}
+
+.news-archive .news-item {
+  padding: 10px 0 0 10px;
+  margin-bottom: 48px;
+  margin-right: 0;
+  width: 100%;
+
+  &:hover,
+  &:focus {
+    .news-item__image {
+      -webkit-filter: none;
+      filter: none;
+    }
+
```

Comment:
This is what we want for accessibility. However, since the `news-item` itself is not the link, it isn't working correctly for keyboard controls. I think we either want to make the news item itself the link, or make sure to add the focus/hover state to the link element as well.

Reply:
That's weird! The news item should be wrapped in a link. Keyboard navigation is working as expected for me on the news page and news archive.